### DISCUSSION TITLE
quad periodic mesh

### DISCUSSION THREAD

**Discussion by shuaizhou001** [neperfepx/neper#680] [Q&A]
Hi Romain,
Is there possible to generate the quad periodic mesh (C3D4 or C3D10) in Neper?
For example, firstly, use this command `**neper -T -n 50 -morpho gg -per all -o xxx**` to generate the model,
after, I need to cut this model and obtain a cube domain (1,1,1). How to do this step?
Finally, use neper -M xxx.tess -rcl 1 -dim all -format inp. `**-dim all** ` is very important for me.

Thank you!

**Comment by rquey** [comment]
The best Neper can do is (using `-elt hex`):
```
neper -T -n 50 -morpho gg -per all -o xxx
neper -M xxx.tess -rcl 0.4 -elt hex
neper -V xxx.msh -print img
```
![image](https://user-images.githubusercontent.com/2659376/232413573-66989318-a124-486d-b25b-6a94a399fa86.png)

However, `-dim all` will generally not be available.

Otherwise, see #98.
